pkgs/by-name/mt/mtk_uart/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  libusb1,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mtk_uartboot";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "981213";
+    repo = "mtk_uartboot";
+    rev = "f2a0977b5775f49cc630f0a18bcec993f3b23975";
```

https://github.com/981213/mtk_uartboot/releases/tag/v0.1.1

```suggestion
    tag = "v${finalAttrs.version}";
```